client/lib/track-element-size/index.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import { useRef, useEffect, useState, useCallback } from 'react';
+import { throttle } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import afterLayoutFlush from 'lib/after-layout-flush';
+
+const THROTTLE_RATE = 200;
+
+// Indexable version of DOMRect / ClientRect.
+interface UseWindowResizeRect {
+	[index: string]: number | undefined;
```

Comment:
This is rather annoying 😞

Reply:
No longer needed, thanks to your suggestion. Thanks!